lib/src/theme/themes/default_theme_no_secondary_border_variant.dart

```diff
@@ -756,6 +756,9 @@ class ShadDefaultThemeNoSecondaryBorderVariant extends ShadThemeVariant {
       tabForegroundColor: colorScheme.foreground,
       tabSelectedForegroundColor: colorScheme.foreground,
       tabSelectedShadows: ShadShadows.sm,
+      tabDecoration: ShadDecoration(
+        border: ShadBorder.all(radius: radius, width: 0),
```

Comment:
Update the radius to 4
<img width="1131" alt="Screenshot 2024-12-02 at 10 41 38" src="https://github.com/user-attachments/assets/f9121fbe-63bd-40fd-b0b9-b9463e5c2be6">

As you can see in the image above, the default radius used by ui.shadcn.com is 4px.
0.5 rem = 8 px
8px - 4px = 4px

Reply:
Just checked the source code and it uses `rounded-sm` which is 2px.
https://github.com/shadcn-ui/ui/blob/805ed4120a6a8ae6f6e9714cbd776e18eeba92c7/apps/www/registry/default/ui/tabs.tsx#L32C66-L32C76

So the source code and what is rendered on the docs is different.
No idea of what is supposed to be the right value.
Probably 4px is more elegant than just 2px